Extend and rewrite ucp troubleshooting

alexmavr opened the pull request:
Signed-off-by: Alex Mavrogiannis <alex.mavrogiannis@docker.com>
### Proposed changes

Rewrite the UCP troubleshooting docs to:
- Include the most common node states
- Emphasize taking a support dump and also a debug-level support dump


This is still a work in progress, but I wanted to get a review started since this is going to be a big set of changes

joaofnfernandes commented on datacenter/ucp/2.1/guides/admin/monitor-and-troubleshoot/index.md:
```diff
@@ -5,66 +5,63 @@ keywords: Docker, UCP, troubleshoot
+`_ping` endpoint, then a detailed error message will be returned if any
+components are unhealthy. 
+
+If you're accessing the `_ping` endpoint through a load balancer, you'll have no
```
This is awesome! We tend to assume that this is obvious, but it's the kind of thing that is only obvious in hindsight. It really helps having this, specially when you're troubleshooting and starting to have tunnel vision.

joaofnfernandes commented:
@alexmavr since this content is so important, I'm getting this ready to go live.
I've cleaned up your proposal, added screenshots for providing more context, and simplified some instructions.

Once the tests pass, we're good to go live 🚀

alexmavr commented:
Thanks Joao, I was planning for what we have here now to be simply a
skeleton of the actual content, but it you think it's so important to get
it out now I can iterate at a follow-up PR

On Feb 25, 2017 3:07 AM, "Joao Fernandes" <notifications@github.com> wrote:

> @alexmavr <https://github.com/alexmavr> since this content is so
> important, I'm getting this ready to go live.
> I've cleaned up your proposal, added screenshots for providing more
> context, and simplified some instructions.
>
> Once the tests pass, we're good to go live 🚀
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/docker/docker.github.io/pull/1843#issuecomment-282451671>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAph-UAl00qPhISJLa2GnhCsM2MpEu6Wks5rf4zugaJpZM4ME6K6>
> .
>
